.github/workflows/lint.yml

```diff
@@ -1,11 +1,22 @@
-name: lint
-on: # rebuild any PRs and main branch changes
+name: TSLint
```

Comment:
I would like to move to ESLint.

Reply:
Yup, I've looked at those migrations before, but never go it right. I think it makes sense, but I don't think I can do it as part of this :)

Reply:
It is better to do it later.